mantaray/views.py

```diff
@@ -156,16 +146,15 @@ def add_notification(self, popup_text: str) -> None:
         self._update_view_selector()
         self.irc_widget.event_generate("<<NotificationCountChanged>>")
         _show_popup(self.view_name, popup_text)
+        playsound("mantaray/audio/notify.mp3")
```

Comment:
Please include a setting in the setting dialog (right-click a server, "Server settings...") for disabling this. Maybe it shouldn't even be on by default? It doesn't have to be channel-specific IMO, just a checkbox that applies to all channels.

Ideally the same setting would apply to all servers, but there is no setting dialog for not-server-specific settings yet, and you don't have to create one just for this PR. (Mantaray supports being connected to multiple servers at once, but there's not yet a GUI for adding more servers.)

Reply:
@Akuli I'm trying to figure out how to get the boolean from config to `class View`where this code is. However, `View` is not instantiated anywhere that I can see with `git grep`, so I cannot add any arguments to the instantiation. Nothing else in `View` seem to be using the config so I'm not sure how to do this.

Reply:
`View` itself is not instantiated, but its subclasses `ServerView` and `ChannelView` are. This confusion is one reason why I don't like inheritance in general, but I found it really makes sense for mantaray's views.

Because this is a server-specific setting, it should be accessed through the server view, e.g. `self.server_view.core.audio_notification`. However, the backend doesn't play audio, so it would make sense to just store the setting in the server view instead of getting the backend involved, as in `self.server_view.audio_notification`. Look at the `extra_notifications` setting for how this is done.